test/ratings.e2e-spec.ts

```diff
@@ -0,0 +1,212 @@
+        .set(authHeader)
+        .expect(HttpStatus.OK);
+    });
+
+    it(`should return ${HttpStatus.OK} and rating object`, async () => {
+      const rating: Rating = await createTestRating();
+      return request(app.getHttpServer())
+        .get(RATINGS_ENDPOINT)
+        .set(authHeader)
+        .expect(HttpStatus.OK)
+        .expect(({ body }) => {
+          expect(Array.isArray(body)).toBe(true);
+          (body as Array<any>).forEach((item) => {
+            expect(item.id).toEqual(rating.id);
+            expect(item.stars).toEqual(rating.stars);
+            expect(item.text).toEqual(rating.text);
+            expect(item.createdAt).toEqual(rating.createdAt.toISOString());
+            expect(item.updatedAt).toEqual(rating.updatedAt.toISOString());
+          });
+        });
+    });
+
+    it(`should return ${HttpStatus.OK} and array of ratings with length of 10`, async () => {
+      for (let i = 0; i < 10; i++) {
+        await createTestRating();
+      }
+
+      return request(app.getHttpServer())
+        .get(RATINGS_ENDPOINT)
+        .set(authHeader)
+        .expect(HttpStatus.OK)
+        .expect(({ body }) => {
+          expect(Array.isArray(body)).toBe(true);
+          expect((body as Array<any>).length).toBe(10);
+        });
+    });
+  });
+
+  describe(RATINGS_ID_ENDPOINT + ' (GET)', () => {
+    it('should exist', async () => {
+      const rating: Rating = await createTestRating();
+      return request(app.getHttpServer())
+        .get(RATINGS_ID_ENDPOINT.replace(':id', rating.id))
+        .expect((response) => response.status !== HttpStatus.NOT_FOUND);
+    });
+
+    it(`should return ${HttpStatus.UNAUTHORIZED} without authorization`, async () => {
+      const rating: Rating = await createTestRating();
+      return request(app.getHttpServer())
+        .get(RATINGS_ID_ENDPOINT.replace(':id', rating.id))
+        .expect(HttpStatus.UNAUTHORIZED)
+        .expect(isErrorResponse);
+    });
+
+    it(`should return ${HttpStatus.OK} with authorization`, async () => {
+      const rating: Rating = await createTestRating();
+      return request(app.getHttpServer())
+        .get(RATINGS_ID_ENDPOINT.replace(':id', rating.id))
+        .set(authHeader)
+        .expect(HttpStatus.OK);
+    });
+
+    it(`should return ${HttpStatus.OK} and rating object`, async () => {
+      const rating: Rating = await createTestRating();
+      return request(app.getHttpServer())
+        .get(RATINGS_ID_ENDPOINT.replace(':id', rating.id))
+        .set(authHeader)
+        .expect(HttpStatus.OK)
+        .expect(({ body }) => {
+          expect(body.id).toEqual(rating.id);
+          expect(body.stars).toEqual(rating.stars);
+          expect(body.text).toEqual(rating.text);
+          expect(body.createdAt).toEqual(rating.createdAt.toISOString());
+          expect(body.updatedAt).toEqual(rating.updatedAt.toISOString());
+        });
+    });
+  });
+
+  describe(RATINGS_ID_ENDPOINT + ' (DELETE)', () => {
```

Comment:
I believe same here, make sure to test for all possible response codes (also NOT_FOUND) and always test for parameter validation (uuid)